package.json

```diff
@@ -28,7 +28,9 @@
     "lint": "npx eslint src --ext .js,.ts,.tsx",
     "lint:fix": "npx eslint --fix src --ext .js,.ts,.tsx",
     "prebuild": "rm -rf dist",
-    "build": "NODE_ENV=production microbundle --name Alma --sourcemap --jsx React.createElement --jsxImportSource react --globals react/jsx-runtime=jsx --define process.env.NODE_ENV=production --css-modules true",
+    "build:minified": "NODE_ENV=production microbundle --name Alma --sourcemap --jsx React.createElement --jsxImportSource react --globals react/jsx-runtime=jsx --define process.env.NODE_ENV=production --css-modules true",
```

Comment:
@PaulRouss3l 

Please add / replace this 2 actions in `.github/workflows/ci-cd.yaml` and remove your commit with `build:raw` (rebase) because this will be done when we will merge into `master` ;)